agent/exec/container/executor.go

```diff
@@ -29,6 +30,7 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 
 	var plugins []api.PluginDescription
 	addPlugins := func(typ string, names []string) {
+		sort.Strings(names)
```

Comment:
I did it this way because we already have a predefined list of plugins (for which `addPlugins` is called explicitly). Your suggestion is obviously more general, and I can switch to that if you think we'll need it in the future.